djangoapp/.gitignore

```diff
@@ -0,0 +1 @@
+*.py[cod]
```

Comment:
We want to ignore python cache files globally, so this should be in the repo root. Also as this is a Python 3 project we should ignore `__pycache__` directories:

```
__pycache__/
*.py[cod]
```

Reply:
Sure - but you _never_ want to check in `.pyc` files, so it should be set globally.